microsoft-edge/devtools-guide-chromium/whats-new/2023/02/devtools-110.md

```diff
@@ -0,0 +1,175 @@
+
+The right-click menu for moving an optional tool between Quick View and the Activity Bar now includes the position of the Activity Bar, such as:
+* **Move to bottom Quick View**
+* **Move to side Quick View**
+* **Move to left Activity Bar**
+* **Move to top Activity Bar**
+
+![Right-click menu showing 'Move to bottom Quick View'](./devtools-110-images/move-to-bottom-quick-view.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Tool name remains visible when you right-click the tool icon
+
+The context menu for moving a tool between the Activity Bar and Quick View no longer obscures the tool's title in the toolbar:
+
+For example, the **Network Conditions** tool name remains visible when you right-click the tool's tab in Quick View:
+
+![The right-click menu in Quick View](./devtools-110-images/focus-mode-moving-tools-context-menu.png)
+
+As another example, the **Network** tool name remains visible when you right-click the tool's tab in the Activity Bar:
+
+![The right-click menu in the Activity Bar](./devtools-110-images/tool-name-remains-visible.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Focus is preserved in Activity Bar or Quick View when moving a tool between them
+
+For keyboard shortcut and assistive technology users, focus is preserved in the Activity Bar or Quick View when moving the currently selected tool from one to the other or removing it.
+<!-- todo: is focus preserved within the tool, or within the toolbar? -->
+
+<!-- todo: screenshot?
+![alt text](./devtools-110-images/filename.png)
+-->
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## Accessibility improvements for themes, customizing locations, and the More Tools button
+
+<!-- Subtitle: Using assistive technology such as screen readers and keyboard shortcuts with DevTools is now easier. -->
+
+The following accessibility issues have been fixed, and using assistive technology such as screen readers and keyboard shortcuts with DevTools is now easier.
+
+
+<!-- ------------------------------ -->
+#### Screen readers correctly report when the More Tools menu is expanded
+
+In previous versions of Microsoft Edge, screen readers reported that the **More tools** (![Plus sign](./devtools-110-images/more-tools-focus-mode-icon.png)) button was collapsed, when the **More tools** menu was actually expanded.  In Microsoft Edge 110, this issue has been fixed.
+
+Screen readers now correctly report when the **More tools** (![Plus sign icon](./devtools-110-images/more-tools-focus-mode-icon.png)) menu is expanded:
+
+![The 'More tools' menu in Focus Mode](./devtools-110-images/a11y-focus-mode-more-tools.png)
+
+See also:
```

Comment:
See my previous comment about the See also links. I don't think we need this 3 times for just this announcement. I think we only need one See also link at the end.